docs/DEV_GUIDE.md

```diff
@@ -33,10 +33,23 @@ summarizer/
 │   │   └── thanos_service.py # Thanos integration
 │   ├── ui/                # Streamlit UI
 │   │   └── ui.py         # Multi-dashboard interface
+│   ├── mcp_server/        # Model Context Protocol server
+│   │   ├── api.py         # MCP API implementation
+│   │   ├── main.py        # HTTP server entrypoint
+│   │   ├── stdio_server.py # STDIO server for AI assistants
+│   │   ├── tools/         # MCP tools (observability_tools.py)
+│   │   └── integrations/  # AI assistant integration configs
 │   └── alerting/          # Alerting service
 │       └── alert_receiver.py # Alert handling
 ├── deploy/helm/           # Helm charts for deployment
+│   ├── mcp-server/        # MCP server Helm chart
+│   ├── metrics-api/       # Metrics API Helm chart
+│   ├── ui/                # UI Helm chart
+│   └── rag/               # RAG components (llama-stack, llm-service)
 ├── tests/                 # Test suite
+│   ├── mcp/               # MCP server tests
```

Comment:
I don't think this correct, we haven't right any mcp server test but let's keep it for future